docs/controls/BladeView.md

```diff
@@ -66,6 +66,12 @@ Here is an example of a BladeView where the `BladeMode` property is binded to a
 
 ```
 
+## AutoCollapseCountThreshold
+
+If you want to use the BladeView for handling a flow of actions, you can use the `AutoCollapseCountThreshold` property to tell it to start auto collapsing BladeItems after a certain threshold count has been reached. This will also help keeping a clean, uncluttered screen real estate.
```

Comment:
Change
>This will also help keeping a clean, uncluttered screen real estate.

To
>This will also help keep a clean, uncluttered screen real estate.

(change keeping to keep)